src/app/body/create-new-organization/team-details/team-details.component.ts

```diff
@@ -106,6 +126,9 @@ export class TeamDetailsComponent implements OnInit {
   async createNewTeamWithLabels() {
     this.teamFormSubmitted.emit({ submitted: true })
     const callable = this.functions.httpsCallable('createNewTeamWithLabels');
+    if(this.selectedTeamId != undefined){
+      this.organizationDomain = this.backendService.getOrganizationDomain();
+    }
```

Comment:
I believe, even if selectedTeamId is not available we still need organizationDomain to create new team.

The code is sent without testing.